components/kyma-environment-broker/internal/provider/aws_provider.go

```diff
@@ -97,7 +97,7 @@ func (p *AWSTrialInput) Defaults() *gqlschema.ClusterConfigInput {
 		GardenerConfig: &gqlschema.GardenerConfigInput{
 			DiskType:       ptr.String("gp2"),
 			VolumeSizeGb:   ptr.Integer(50),
-			MachineType:    "m4.xlarge",
+			MachineType:    "m5.xlarge",
```

Comment:
@piotrmiskiewicz : QQ - I haven't found this machine type in the schema (wasn't there before, see https://github.com/kyma-project/control-plane/pull/644/files#diff-b878f19cc19576376ee78a3ae7f507952afaf2e339968a132ea4914301dfc470R169). Could that become an issue?

Reply:
yes, the line you mentioned, contains the machine types in the schema

Reply:
Yes, but currently the machine type we use for trials is missing there (`m5.xlarge"`). Should I add this type to the schema too?